events/viewsv2.py

```diff
@@ -8,7 +8,7 @@
 import re
 from django.conf import settings
 
-from config import CHANNEL_KEY
+from spoken.config import CHANNEL_KEY
```

Comment:
Please revert back to original one,  which is `from config import CHANNEL_KEY`

Reply:
Ok